docker/server/.env.example

```
@@ -2,6 +2,9 @@
 
 NEXT_PUBLIC_HEXCLAVE_API_URL=http://localhost:8102
 NEXT_PUBLIC_HEXCLAVE_DASHBOARD_URL=http://localhost:8101
+# Set this only when the independent /tv app is served from a different origin.
+# It enables credentialed TV API requests exclusively for that exact origin.
+HEXCLAVE_TV_DISPLAY_ORIGIN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HEXCLAVE_TV_DISPLAY_ORIGIN key should go before the NEXT_PUBLIC_HEXCLAVE_API_URL key

(UnorderedKey)